docs/hardware-requirements.md

```diff
@@ -58,3 +58,65 @@ There are a few known hardware related issues with illumos.
 - There have been several issues with Intel CPUs regarding
   their C-States. SmartOS has worked around them, but you should
   consider disabling them in your BIOS.
+
+SmartOS/Triton
+
+- SmartOS depends upon the hardware bios serial number in order to
+  generate a UUID on boot; which is then assigned to a given compute
+  node. (As displayed by the `sysinfo` command) In some rare cases, such
+  as with the "Dell PowerEdge c6100" blade-type line of servers, the
+  chassis sometimes incorrectly assigns the same serial number across
+  all of the blades installed in the same unit. This can cause issues
+  for some software such as "Triton Datacenter" that directly rely upon
+  the chassis serial number and the subsequent UUID generated on boot
+  by SmartOS, to be entirely unique.
+
+  Example:
+
+  In the case of "Triton Data Center" and the aforementioned, "Dell
+  c6100" chassis, 1 compute node (blade/sled), will be properly detected
+  by "cnapi" and consequently, the "Operator Portal" on boot, while the
+  other 3 quietly PXE boot without detection by Triton. To determine if
+  a duplicate server UUID is the cause of your issue, simply ssh into
+  each of the compute nodes in question, and run: `sysinfo | json UUID`.
+  If more than 1 compute node share the same UUID, then a duplicate
+  serial number is likely the cause of the issue. You can also verify
+  the duplicate serial numbers on each node with the following:
+
+  `ipmitool fru print 0`
+
+  You should receive output resembling this:  
+
+    Chassis Type          : Rack Mount Chassis
+    Chassis Part Number   :
+    Chassis Serial        :  
+    Board Mfg Date        : Wed Nov  7 02:43:00 2012
+    Board Mfg             : Dell Inc.
+    Board Product         : PowerEdge  
+    Board Serial          : CN0D61XP747512B60255A08
+    Board Part Number     : 282BNP0616
+    Product Manufacturer  : Dell Inc.
+    Product Name          : C6100
+    Product Part Number   :
+    Product Version       :
+    Product Serial        : DB3KYV1
+    Product Asset Tag:
```

Comment:
It must be something with the browser cache when reviewing with 'make serve'. They were all code blocks but refreshes would break the formatting. Ill put them back in.